recipe/meta.yaml

```diff
@@ -23,25 +23,27 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cffi                                   # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - llvm-openmp  # [osx]
+    - libgomp      # [linux]
     - ccache
+
   host:
     - python
+    - numpy
+    - pybind11 >=2.2
     - pip
-    - cffi
-    - llvm-openmp  # [osx]
+
   run:
     - python
-    - numpy
-    - cffi
+    - {{ pin_compatible('numpy') }}
+    - pybind11 >=2.2
```

Comment:
I thought pybind11 was header-only at build time. Does it have a runtime bit I am not remembering?

Reply:
Not that I know of.  I was just copying from the GalSim recipe.  https://github.com/conda-forge/galsim-feedstock/blob/main/recipe/meta.yaml

Reply:
I just confirmed that if I remove pybind11 from my conda environment, TreeCorr still runs fine, so yeah, I think we can remove this.
```suggestion
```